crates/runtime/src/libcalls.rs

```diff
@@ -189,6 +189,13 @@ pub mod trampolines {
             self
         }
     }
+
+    impl LibcallResult for u64 {
```

Comment:
This new trait implementation is gone now anyway, because we changed `tc_resume` to return a `TrapReason` in the meantime.
Just for completeness: Note that the conversion between `SwitchDirection` (nee `SwitchReason) and `u64` was never happening here, but always inside `tc_resume`. The libcall infrastructure itself just passes the `Ok` value on without further interpretation.